build.gradle

```diff
@@ -44,27 +47,32 @@ run {
 // ### Checkstyle plugin settings
 checkstyle {
     // Latest version at https://github.com/checkstyle/checkstyle/tags
-    toolVersion = '10.12.1'
+    toolVersion = '10.23.1'
+
     // Keep the below file updated along with subsequent versions of Checkstyle (make sure to choose
     // the tag matching the toolVersion above):
     // https://github.com/checkstyle/checkstyle/blob/checkstyle-10.3.2/src/main/resources/google_checks.xml
+
     // NOTE: After updating the local version of config/checkstyle/google_checks.xml again in the
-    // future, it may be necessary to subsequently replace the following line in the local copy to
-    // prevent suppressions from being ignored:
+    // future, you will need to make two manual changes:
+    //
+    // 1. Allow relative path for suppressions
```

Comment:
Nit: 1. Replace the following line in the local copy to prevent suppressions from being ignored: